### DISCUSSION TITLE
How to disable `unfetch` polyfill?

### DISCUSSION THREAD

**Discussion by antoinerey** [nuxt/nuxt#10116] [Questions (Nuxt 2)] [answered]
We're using `axios` to manage HTTP requests both client and server-side, and never use `fetch` in any way. 

However, we do still load a polyfill (`unfetch`) for legacy browsers to all our users. I understand it's not that big ([560 bytes on Bundlephobia](https://bundlephobia.com/package/unfetch@4.2.0)), but still, it's unused code that must be downloaded, parsed, and executed.

Hence my question: **Is there any way to disable that polyfill?** I've search for it into the documentation, but no luck. 🙏

**Comment by pi0** [comment, marked as answer]
Hi. Yes you can disable polyfill using `fetch: { client: false }` in nuxt.config. Also when using nuxt-bridge, this is the default behavior. Read more: https://v3.nuxtjs.org/getting-started/bridge

**Reply by antoinerey** [reply]
Wonderful! Thanks a lot for your answer. 🙇 

If I may, is there some documentation about this, or more globally about features that we may want to disable to save a few bytes from our users? 🙏

**Reply by pi0** [reply]
You can use this file as a reference: https://github.com/nuxt/nuxt.js/blob/dev/packages/config/src/config/_app.js#L60

In nuxt3, features are documented and most of them will be opt-in to be enabled by usage
